comp/game/robot-flag-mounts.yaml

```diff
@@ -0,0 +1,20 @@
+summary: Prepare robot flags for the competition
```

Comment:
This title doesn't seem to match the filename, it also seems to just be a duplicate of `comp/game/robot-flags.yaml`. Should this be something else?